ui/app/components/modals/cancel-transaction/cancel-transaction.container.js

```diff
@@ -0,0 +1,55 @@
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+import R from 'ramda'
+import { multiplyCurrencies } from '../../../conversion-util'
+import { bnToHex } from '../../../helpers/conversions.util'
+import withModalProps from '../../../higher-order-components/with-modal-props'
+import CancelTransaction from './cancel-transaction.component'
+import { showModal, hideModal, createCancelTransaction } from '../../../actions'
+
+const mapStateToProps = (state, ownProps) => {
+  const { metamask } = state
+  const { transactionId, originalGasPrice } = ownProps
+  const { selectedAddressTxList } = metamask
+  const transaction = R.find(({ id }) => id === transactionId)(selectedAddressTxList)
+  const transactionStatus = transaction ? transaction.status : ''
+
+  const defaultNewGasPrice = bnToHex(multiplyCurrencies(originalGasPrice, 1.1))
```

Comment:
`bnToHex(multiplyCurrencies(originalGasPrice, 1.1))` should probably hex prefix `originalGasPrice`, or specify the base to `multiplyCurrencies`. Otherwise, there is a possibility that `originalGasPrice` will be all numeric digits and the `multiplyCurrencies` call will treat it as a decimal

Also, instead of using `bnToHex`, a config object can be passed to `multiplyCurrencies`, with the `toNumericBase` property specified.

So this would become:
```
multiplyCurrencies(originalGasPrice, 1.1, {
    toNumericBase: 'hex',
    multiplicandBase: 16,
    multiplierBase: 10,
  })
```

I guess then it could be moved into something like a `increaseByTenPercent` helper function